docs/include/filter.asciidoc

```diff
@@ -0,0 +1,171 @@
+[id="plugins-{type}s-common-options"]
+==== Common Options
+
+The following configuration options are supported by all filter plugins:
+
+[cols="<,<,<",options="header",]
+|=======================================================================
+|Setting |Input type|Required
+| <<plugins-{type}s-{plugin}-add_field>> |<<hash,hash>>|No
+| <<plugins-{type}s-{plugin}-add_tag>> |<<array,array>>|No
+| <<plugins-{type}s-{plugin}-enable_metric>> |<<boolean,boolean>>|No
```

Comment:
Nevermind I totally misread one of the files here. I read the codecs doc but assumed it was some 'base plugin' doc.

Disregard my comment above